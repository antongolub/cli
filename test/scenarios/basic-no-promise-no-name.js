'use strict'
require('../..')(() => {
  console.log('started')
  console.log('finished')
})
