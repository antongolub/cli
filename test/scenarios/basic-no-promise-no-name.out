started
finished
Error: Application entry point did not return a promise.
1
